core/src/test/java/io/debezium/connector/cassandra/FileOffsetWriterTest.java

```diff
@@ -107,6 +107,13 @@ public void testFlush() throws IOException {
         process(commitLogRecordDiffTable);
 
         offsetWriter.flush();
+        // Sleep a little bit to ensure the operation is done
+        try {
+            Thread.sleep(100);
```

Comment:
This is a brittle approach and can lead to flaky tests. Please use a more robust solution based for example on `Awaitility`.

Reply:
I change OffsetWriter methods to return Future object to use async approaches.